pages/organization/students/index.tsx

```diff
@@ -114,14 +116,23 @@ const Students: NextPage<IStemServices.IProps, IStemServices.InitialProps> = (
   };
   const toggleModal = () => {
     setStudentModal(!studentModal);
+    router.push({
+      pathname: "/organization/students/studentDetails",
+      query: {
```

Comment:
instead of passing tinstead of passing the whole student details, just pass the ID of the student as query a parameter and fetch the data from he whole student details, just pass the ID of the student as query a parameter and fetch the data inside the useEffect